gen_cert.py

```diff
@@ -102,7 +102,9 @@ class CertificateGen(object):
     """Manages the pdf, signatures, and S3 bucket for course certificates."""
 
     def __init__(self, course_id, template_pdf=None, aws_id=None, aws_key=None,
-                 dir_prefix=None, long_org=None, long_course=None, issued_date=None):
+                 dir_prefix=None, long_org=None, long_course=None, issued_date=None,
+        title=None,
```

Comment:
@stvstnfrd I have a vim plugin so I can see issues as I introduce them.  I thought my settings could be off, most the violations are line-too-long(not a big deal), but there are 11 unused imports.